oat-sa/extension-tao-delivery
# Feature/TR-5999/Design the waiting page

### Related to: https://oat-sa.atlassian.net/browse/TR-5940

### Summary

Apply the correct design to the waiting page.

<img width="941" alt="image" src="https://github.com/oat-sa/extension-tao-delivery/assets/1500098/f7f2f7b9-04d4-41a3-ba9a-ba8c8875847a">

### Details
Use the same design as for the list boxes used un the list of tests.

### How to test
- check out the branch: `git checkout -t origin/feature/TR-5999/waiting-page`
- log in a test taker
- start a test
- replace the URL query string with `?waitingPage=1` (after the `/taoDelivery/DeliveryServer/runDeliveryExecution`)
- to make the action redirect to the correct page, you can also copy the previous URL and encode it as query parameter, then add the parameter `&deliveryExecutionUrl=THE_URL` (replacing `THE_URL` with the URL).

Note: to encode a value as a query parameter, open the browser console and type `encodeURIComponent('THE_URL')`, replacing `THE_URL` with the URL. It will display the encoded value :)
